src/frontend/src/common/queries/auth.ts

```diff
@@ -34,6 +34,8 @@ export const mapUserData = (obj: RawUserRequest): User => {
   const mappedUserData: User = {
     acknowledgedPolicyVersion: obj.acknowledged_policy_version,
     agreedToTos: obj.agreed_to_tos,
+    // @ts-expect-error TODO (mlila): types
+    email: obj.email,
```

Comment:
is this @ts-expect-error supposed to be here?

Reply:
yes, otherwise the type checker fails. That's why it is a todo item.